src/javascript/binary/websocket_pages/user/account/settings/personal_details.js

```diff
@@ -311,6 +311,18 @@ const PersonalDetails = (() => {
     };
 
     const onLoad = () => {
+        $(() => {
+            $('#form_addressline1_message').accordion({
+                heightStyle: 'content',
+                collapsible: true,
+                active     : false,
+            });
+            $('#form_addressline2_message').accordion({
+                heightStyle: 'content',
+                collapsible: true,
+                active     : false,
+            });
+        });
```

Comment:
instead of repeating the same lines in all these files we can add it in a common file and call the function everyone, probably account_opening.js under common_functions folder would be a good place